public/assets/js/index.js

```diff
@@ -0,0 +1,5 @@
+var now = new Date()
```

Comment:
What about?

```js
var year = new Date().getFullYear()
```

Reply:
You should use `let` instead of `var`

Reply:
We are not using babel. SO it's better to leave it `var` no harms here :)

Reply:
I returned the `var`